### PR TITLE
Improve benefits in Handbook

### PR DESCRIPTION
**Pull request by lukemorton**
## What

- [x] Highlight some elements of Staff Welfare as Benefits in Handbook
- [x] Add transparent salary bands and annual salary reviews to benefits
- [x] Add Learn Tech to benefits
- [x] Add expenses paid to benefits

## Why

We should be loud and proud about the reasons for working at Made Tech. We are awesome!

### CODE REVIEW

**Comment by yaseminercan** [comment]
I can't go into the sections to read or did you only change the menu order?

**Comment by davidwinter** [comment]
Nit: sentence case for the benefit headings

**Comment by davidwinter** [comment]
Should this also be reflected on the primary `README.md`?

https://github.com/madetech/handbook

File: README.md [hunk 1]
```diff
@@ -41,17 +41,31 @@ Contributing to the handbook isn't only possible for `git` users, find steps to
...
 ### Benefits & Perks
 
+Balancing life and work:
+
 * [Flexible Holiday](benefits/flexible_holiday.md) – We trust you to take as much holiday as you need
 * [Flexible Working Hours](benefits/working_hours.md) – We are flexible with what hours you work
 * [Flexible Working Days](benefits/flexible_working.md) – We are flexible to the amount of days you work in a week
-* [Pension Scheme](benefits/pension_scheme.md) – We provide a pension scheme so you can save for your future and we'll contribute to it
-* [Friday Lunches](benefits/friday_lunch.md) – We send a random 8 of our team out to lunch every Friday so everyone can get to know each other
+* [Flexible Parental Leave](guides/welfare/parental_leave.md) – We provide flexible parental leave options
```
Comment (kjdchapman) [inline README.md]: We could link topics as part of a sentence:

```suggestion
* We provide [flexible leave for new parents](guides/welfare/parental_leave.md).
```

Renders as:
> We provide [flexible leave for new parents](link).

File: README.md [hunk 2]
```diff
@@ -41,17 +41,31 @@ Contributing to the handbook isn't only possible for `git` users, find steps to
...
 * [Remote Working](benefits/remote_working.md) – We offer part-time remote working for all our staff
+* [Paid counselling](guides/welfare/employee_assistance.md) – We offer paid counselling as well as financial and legal advice
+* [Paid anniversary break](benefits/paid_anniversary_break.md) – We celebrate your 3 and 5 year anniversary at Made Tech by buying your family a holiday
+
+Making work as fabulous as possible:
+
+* [Work Ready](benefits/work_ready.md) – We'll buy you a Macbook, ergonomic equipment, books, conferences, training, and more
+* [Learn Tech](learning/readme.md) – We spend every Friday afternoon learning rather than working
+* [Friday Lunches](benefits/friday_lunch.md) – We randomly match up 8 colleagues every Friday and pay for lunch
```
Comment (kjdchapman) [inline README.md]: If we're linking mid-sentence, we could combine some related sentences into one

> We take eight lucky people out for a [paid lunch each Friday daytime](link), and [pay for social drinks each Friday evening](link).

File: README.md (hunk 1 above)
Comment (lukemorton) [reply README.md]: Let's handle this in a follow up

File: README.md (hunk 2 above)
Comment (lukemorton) [reply README.md]: Yeah I like it, although at a glance might not be as readable?

**Pull request merged**